README.md

```diff
@@ -12,7 +12,11 @@ To preview, run these commands in your terminal (as long as you’ve previously
 3. npm install
 4. npm install -g @angular/cli
 5. npm i -g typescript
-6. ng serve -o
+6. npm install express --save
```

Comment:
it looks like there's a package.json file created for this project and is updated with these packages (express, mongoose and body-parser) - so they don't need to be added to the readme. If you run npm install it should install them.

Reply:
Ok Sure! :)
Will look into that and create a separate issue for updating the package.json file. :)

Reply:
Ok Sure! Will create a separate issue for updating the package.json file. :)

Reply:
Hi @BaaniLeen - you don't need to create a new issue because the package.json already has the packages listed, you just need to remove the lines from the README.md file telling you to install the packages - it's these lines you should remove from the README.md because they are not needed:
`npm install express --save`
`npm install mongoose -- save`
`npm install body-parser --save`

Reply:
Oh okay! Will do that right away. :)
Interesting. @bethanyr Does package.json update on its own when I run an npm install command in my repo? Like I did not update it, explicitly.